Connect4New/Classes/Cell.cs

```diff
@@ -4,7 +4,7 @@ internal class Cell
     {
         #region Public Constructors
 
-        public Cell(int column, int line)
```

Comment:
Why do you still have this class if you created Connect4Cell class?

Reply:
I don't know. That class does not appear in my project.